diff --git a/BUILD.gn b/BUILD.gn
index 2068409ddd..107cb570e3 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -3358,21 +3358,3 @@ if (skia_enable_tools) {
   }
 }
 
-if (is_ios && skia_use_metal && skia_enable_gpu &&
-    !skia_enable_flutter_defines) {
-  group("minimal_ios_mtl_skia_app") {
-    deps = [ "experimental/minimal_ios_mtl_skia_app" ]
-  }
-}
-
-if (is_ios && skia_enable_skottie && !skia_enable_flutter_defines) {
-  group("skottie_ios") {
-    deps = [ "tools/skottie_ios_app" ]
-  }
-}
-
-skia_executable("skia_c_api_example") {
-  sources = [ "experimental/c-api-example/skia-c-example.c" ]
-  include_dirs = [ "." ]
-  deps = [ ":skia" ]
-}
diff --git a/gn/toolchain/BUILD.gn b/gn/toolchain/BUILD.gn
index 8029086eff..47eeeca061 100644
--- a/gn/toolchain/BUILD.gn
+++ b/gn/toolchain/BUILD.gn
@@ -86,7 +86,7 @@ toolchain("msvc") {
   if (target_cpu == "x86") {
     # Toolchain asset includes a script that configures for x86 building.
     # We don't support x86 builds with local MSVC installations.
-    env_setup = "$shell $win_sdk/bin/SetEnv.cmd /x86 && "
+    # env_setup = "$shell $win_sdk/bin/SetEnv.cmd /x86 && "
   } else if (target_cpu == "arm64") {
     # ARM64 compiler is incomplete - it relies on DLLs located in the host toolchain directory.
     env_setup = "$shell set \"PATH=%PATH%;$win_vc\\Tools\\MSVC\\$win_toolchain_version\\bin\\HostX64\\x64\" && "
diff --git a/modules/skparagraph/BUILD.gn b/modules/skparagraph/BUILD.gn
index 45da5fb297..19c5fed5bd 100644
--- a/modules/skparagraph/BUILD.gn
+++ b/modules/skparagraph/BUILD.gn
@@ -39,6 +39,7 @@ if (skia_enable_skparagraph && skia_enable_skshaper && skia_use_icu &&
   if (defined(is_skia_standalone) && skia_enable_tools) {
     skia_source_set("utils") {
       import("skparagraph.gni")
+      complete_static_lib = false
       public_configs = [ ":utils_config" ]
       configs = [ "../../:skia_private" ]
       sources = skparagraph_utils
diff --git a/modules/svg/include/SkSVGDOM.h b/modules/svg/include/SkSVGDOM.h
index 4c2c9d98de..b97c999c3e 100644
--- a/modules/svg/include/SkSVGDOM.h
+++ b/modules/svg/include/SkSVGDOM.h
@@ -14,6 +14,7 @@
 #include "include/private/SkTemplates.h"
 #include "modules/skresources/include/SkResources.h"
 #include "modules/svg/include/SkSVGIDMapper.h"
+#include "modules/svg/include/SkSVGDefs.h"
 
 class SkCanvas;
 class SkDOM;
