diff --git a/BUILD.gn b/BUILD.gn
index e7a79b9e7c..dc3e51e59e 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -3084,15 +3084,3 @@ if (skia_enable_tools) {
     }
   }
 }
-
-if (is_ios && skia_use_metal && !skia_enable_flutter_defines) {
-  group("minimal_ios_mtl_skia_app") {
-    deps = [ "experimental/minimal_ios_mtl_skia_app" ]
-  }
-}
-
-if (is_ios && skia_enable_skottie && !skia_enable_flutter_defines) {
-  group("skottie_ios") {
-    deps = [ "tools/skottie_ios_app" ]
-  }
-}
diff --git a/gn/toolchain/BUILD.gn b/gn/toolchain/BUILD.gn
index e8e6ce4603..804838d3d0 100644
--- a/gn/toolchain/BUILD.gn
+++ b/gn/toolchain/BUILD.gn
@@ -81,7 +81,7 @@ toolchain("msvc") {
   if (target_cpu == "x86") {
     # Toolchain asset includes a script that configures for x86 building.
     # We don't support x86 builds with local MSVC installations.
-    env_setup = "$shell $win_sdk/bin/SetEnv.cmd /x86 && "
+    # env_setup = "$shell $win_sdk/bin/SetEnv.cmd /x86 && "
   } else if (target_cpu == "arm64") {
     # ARM64 compiler is incomplete - it relies on DLLs located in the host toolchain directory.
     env_setup = "$shell set \"PATH=%PATH%;$win_vc\\Tools\\MSVC\\$win_toolchain_version\\bin\\HostX64\\x64\" && "
diff --git a/modules/skparagraph/BUILD.gn b/modules/skparagraph/BUILD.gn
index e8a8b48df6..7859ff97e7 100644
--- a/modules/skparagraph/BUILD.gn
+++ b/modules/skparagraph/BUILD.gn
@@ -21,6 +21,7 @@ if (skia_enable_skparagraph && skia_enable_skshaper && skia_use_icu &&
 
   component("skparagraph") {
     import("skparagraph.gni")
+    complete_static_lib = false
     public_configs = [ ":public_config" ]
     public = skparagraph_public
     sources = skparagraph_sources
