diff --git a/BUILD.gn b/BUILD.gn
index b278798ea9..fc345a2239 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -3376,13 +3376,13 @@ if (skia_enable_tools) {
   }
 }
 
-if (is_ios && skia_use_metal && skia_enable_ganesh) {
+if (is_ios && skia_use_metal && skia_enable_ganesh && skia_build_ios_apps) {
   group("minimal_ios_mtl_skia_app") {
     deps = [ "experimental/minimal_ios_mtl_skia_app" ]
   }
 }
 
-if (is_ios && skia_enable_skottie) {
+if (is_ios && skia_enable_skottie && skia_build_ios_apps) {
   group("skottie_ios") {
     deps = [ "tools/skottie_ios_app" ]
   }
diff --git a/gn/skia.gni b/gn/skia.gni
index 95361cbfdb..0e804d92cf 100644
--- a/gn/skia.gni
+++ b/gn/skia.gni
@@ -100,6 +100,8 @@ declare_args() {
     skia_ios_use_signing = !ios_use_simulator
   }
 
+  skia_build_ios_apps = false
+
   skia_build_rust_targets = false
 }
 
